lib/fluent/plugin/kubernetes_metadata_common.rb

```diff
@@ -54,10 +54,28 @@ def parse_pod_metadata(pod_object)
         self.de_dot!(labels)
         self.de_dot!(annotations)
       end
+
+      # collect container informations
+      container_meta = {}
+      begin
+        pod_object['status']['containerStatuses'].each do|container_status|
+          container_meta[container_status['containerID']] = {
+              'name' => container_status['name'],
+              'image' => container_status['image'],
+              'image_id' => container_status['imageID']
+          }
+        end
+      rescue
+        pod_name = pod_object['metadata']['name']
```

Comment:
I would recommend wrapping this in a `log.debug?` since the only time you would need to retrieve this info is when logging the debug message.